File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int answer1 = 0;
        int answer2 = 0;
        int num1 = 10;
        int num2 = 10;
        foreach(var item in num_list)
        {
            if(item % 2 == 0)//짝수
            {
                answer1 *= num1;
                answer1 += item;
            }
            else//홀수
            {
                answer2 *= num2;
                answer2 += item;
            }
        }
        answer = answer1 + answer2;
        return answer;
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    string oddstring = "";
    string evenstring = "";
    for(auto num : num_list)
    {
        
        if(num % 2 == 0)
            evenstring += to_string(num);
        else
            oddstring += to_string(num);
    }
    return answer = stoi(evenstring) + stoi(oddstring);
}